test/WebSites/ConnegWebSite/Controllers/ProducesContentBaseController.cs

```diff
@@ -37,6 +37,6 @@ public virtual string ReturnClassNameWithNoContentTypeOnAction()
         public virtual string ReturnClassNameContentTypeOnDerivedAction()
         {
             return "ProducesContentBaseController";
-        }
+        }        
```

Comment:
I am trying to get rid of this and next 3 files where I have reverted back the changes.